packages/tables/docs/03-columns/01-getting-started.md

```diff
@@ -386,6 +386,26 @@ TextColumn::make('title')
     ->tooltip(fn (Model $record): string => "By {$record->author->name}")
 ```
 
+## Alignment
+
+The alignment of columns can be customized by using methods like `alignRight()`:
```

Comment:
Table columns are aligned to the start by default. You may change the alignment using the `alignment()` method and passing it `start`, `center`, `end`, `left`, `right` or `justify`: